pkg/backend/httpstate/client/client.go

```diff
@@ -1147,6 +1147,10 @@ func getNaturalLanguageSearchPath(orgName string) string {
 	return fmt.Sprintf("/api/orgs/%s/search/resources/parse", url.PathEscape(orgName))
 }
 
+func getPulumiOrgSearchPath(orgName string) string {
+	return fmt.Sprintf("https://app.pulumi.com/%s/resources", url.PathEscape(orgName))
```

Comment:
Should this not take the client address into account for self-hosted customers?

Reply:
It should! I was actually unaware we had that information available here. Let me do some digging and plumb that in.

Reply:
The only place I could see this elsewhere was as passed as a CLI argument to `--client`. Is there another place this information is available? (Backend has the URL of the API we're interacting with, but this URL redirects to the front-end).

Reply:
Ah yeh I don't think we record the frontend, and doing that nicely probably requires service to give a new endpoint for us to call (because we only get the api url from the user when logging in).
Can you just add a TODO to that effect here.